ios/fastlane/Fastfile

```diff
@@ -102,24 +102,10 @@ platform :ios do
       uses_non_exempt_encryption: false
     )
 
+    upload_symbols_to_crashlytics
```

Comment:
providing the path is not needed?

Reply:
It should be automagic, as it was before. I can double check the latest beta, which included this commit.

Reply:
I'm curios if it works. How I understood this works is that you're supposed to download the dsym file, then upload it, but in this case I'm not sure it knows where to get the file from without providing the path.